### DISCUSSION TITLE
Mouse Stuck to Top Half of Screen on NVIDIA 555 - NixOS Wayland

### DISCUSSION THREAD

**Discussion by A1029384756** [ppy/osu#28377] [Q&A]
### Type

Game behaviour

### Bug description

When launching a beatmap (not in menus), the mouse is stuck to the upper half of the screen and cannot move below. (Logs from a different run). Cannot reproduce on Nvidia 550.

### Screenshots or videos

https://github.com/ppy/osu/assets/35206453/217eaf25-5f40-4fc6-b5d1-857168b3bdf3

### Version

2024.521.2

### Logs

[logs.zip](https://github.com/user-attachments/files/15523386/logs.zip)

**Comment by smoogipoo** [comment]
Issues related to graphics drivers are out of our control.
